src/cloud-api-adaptor/podvm/README.md

```diff
@@ -77,31 +77,17 @@ $ docker build -t podvm_builder \
 
 ## Building the image containing the podvm binaries
 
-In order to build the binaries you should be using the corresponding dockerfile
```

Comment:
Hi @beraldoleal !

I like the idea of suggesting the use of make targets wherever possible. IMHO CoCo and kata docs make a poor job on standardizing around Make, several places use call to scripts (or sequence of commands) where you could be using an equivalent make target. Anyway...

However, for this particular change I'm afraid that dropping these lines that explain the relation between builder image and podvm binaries will result on less valuable information for the devs (essentially now we are saying devs to look at the sources). For example, if a dev built the builder image as indicated on the previous section, and he/she run `make podvm-binaries` it won't be pulling that image (instead it's getting  the default quay.io/confidential-containers/podvm-builder-ubuntu). If you see the sections on this doc then you will notice they are chained to show how a dev can build the builder -> binaries -> podvm images.

Reply:
Yes, agree is a little bit confusing. Ideally all the steps should be pointing to the makefile. I have changed only here to fix a broken instruction, for now.

I will rephrase my last paragraph and/or add note to make it clear. Thanks @wainersm